scripts/infra/setup-aws.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env -S bash -e
```

Comment:
What if the script is named something like `local-setup`.  And the command was something like `ensure-aws-system-pkgs`.  So you could support other infras.

Reply:
I like it!